pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationStatusTest.java

```diff
@@ -30,6 +30,7 @@
 import org.broadinstitute.dsm.model.nonpepperkit.NonPepperStatusKitService;
```

Comment:
the changes here are only name changes from `JuniperTestSetupUtil` to `JuniperKitUtil`